app/views/teaching_events/about_git_events.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= render Campaign::HeaderComponent.new(
+  title: ["Get Into Teaching", "events"],
+  colour: "yellow",
+  image: "media/images/content/hero-images/0025.jpg"
+) %>
+
+<div class="subject-specific">
+  <article>
+    <section class="statement">
+      <span class="pink xlarge bold">What to expect</span>
+    </section>
+
+    <section>
+      <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+
+      <p>They're a series of relaxed and informative events giving you the opportunity to get expert advice and talk to teachers and local training providers.</p>
```

Comment:
```suggestion
      <p>These relaxed and informative events give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
```

@GDW7 I wonder if we should just simplify this - the fact that it is a series isn't really of interest to people interested in teaching, I think?

Reply:
@GDW7 The Events team also flagged this as something to change - they don't want to imply that you have to go to more than one!